dev/com.ibm.jbatch.container/src/com/ibm/jbatch/container/services/impl/JPAPersistenceManagerImpl.java

```diff
@@ -2593,12 +2595,15 @@ protected void rollbackIfNewTranWasStarted(Throwable caughtThrowable) throws Per
             }
 
             // If we haven't gotten a new exception to chain, throw back the original one passed in as a parameter
+            logger.fine("No exception encountered, throwing new PersistenceException with original exception chained");
             throw new PersistenceException(caughtThrowable);
         }
 
         protected void resumeAnyExistingLTC() {
+            logger.fine("Will resume any LTC");
```

Comment:
not needed.. entry/exit is enough here.  Don't care either way about the next one.